vue/src/TestResultsPage/TestResultsPage.vue

```diff
@@ -12,7 +12,7 @@
       <p>{{ translate('SecurityInfo_PluginDescription') }}</p>
       <p>
         Learn more: read our guide
-        <a rel='noreferrer' target='_blank' href='https://matomo.org/docs/security/'>
+        <a rel='noreferrer' target='_blank' href='{{ externalRawLink("https://matomo.org/docs/security/") }}'>
```

Comment:
@bx80 this should be :href and need more changes, I tried fixing quickly but would require more work

Reply:
Thanks for spotting this :+1: The same issue exists in a few other places. I've pushed a commit to this PR which should sort this.